chartiq/src/main/java/com/chartiq/sdk/scriptmanager/ChartIQScriptManager.kt

```diff
@@ -85,8 +85,8 @@ internal class ChartIQScriptManager : ScriptManager {
     override fun getSetChartScaleScript(scale: String): String =
         CHART_IQ_JS_OBJECT + "setChartScale(\"$scale\");"
 
-    override fun getAddStudyScript(studyName: String): String =
-        MOBILE_BRIDGE_NAME_SPACE + "addStudy(\"$studyName\");"
+    override fun getAddStudyScript(studyName: String, inputs: String, outputs: String, parameters: String): String =
+        MOBILE_BRIDGE_NAME_SPACE + "addStudy(\"$studyName\", '$inputs', '$outputs', '$parameters');"
```

Comment:
Any reason for the mismatched quotation types?

Reply:
I need the ticks to escape the quotes in the json. I guess studyName can be surrounded by tick marks easy enough.